### ISSUE TITLE
EmailSender QueueAsync send mail to multiple addresses causes validation error

### ISSUE THREAD

**Issue by ardenyy** [abpframework/abp#21719]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

When using IEmailSender and sending mails to mutiple addresses joined by ",", .SendAsync() methods works without a problem, but when .QueueAsync() is used, it causes a email validation error.

Looks like validate email address method only used when the mail is queued and the validation method can not handle multiple addresses.
[Email Sender Base Class](https://github.com/abpframework/abp/blob/dev/framework/src/Volo.Abp.Emailing/Volo/Abp/Emailing/EmailSenderBase.cs)

### Reproduction Steps

trying to send email to multiple addresses with IEmailSender.QeueuAsync() method

### Expected behavior

Validate Email method should be called both send and queue methods and it should handle multiple addresses as valid

### Actual behavior

Volo.Abp.Emailing.EmailSenderBase.ValidateEmailAddress throws ArgumentException: email address "" is not valid!

### Version

8.3.3

### User Interface

Common (Default)

### Database Provider

EF Core (Default)

### Tiered or separate authentication server

None (Default)

### Operation System

Windows (Default)

**Comment by maliming** [comment]
hi

Please share the code to reproduce.

Thanks.

**Comment by ardenyy** [comment]
You can inject IEmailSender to anywhere you want and call its SendAsync and QueueAsync methods with whatever body you want, but "To" parameter must be multiple email addresses joined with "," for example: "example@abp.io,support@abp.io", SendAsync method will work fine while QeueuAsync will throw exception because of validation.

[EmailSenderBase.cs QueueAsync()](https://github.com/abpframework/abp/blob/dev/framework/src/Volo.Abp.Emailing/Volo/Abp/Emailing/EmailSenderBase.cs#L93) 

```csharp
  public virtual async Task QueueAsync(string to, string subject, string body, bool isBodyHtml = true, AdditionalEmailSendingArgs? additionalEmailSendingArgs = null)
  {
      // you can see validation method here, but it wont be called from sendAsync, only from queueAsync
      ValidateEmailAddress(to);

      if (!BackgroundJobManager.IsAvailable())
      {
          await SendAsync(to, subject, body, isBodyHtml, additionalEmailSendingArgs);
          return;
      }

      await BackgroundJobManager.EnqueueAsync(
          new BackgroundEmailSendingJobArgs
          {
              TenantId = CurrentTenant.Id,
              To = to,
              Subject = subject,
              Body = body,
              IsBodyHtml = isBodyHtml,
              AdditionalEmailSendingArgs = additionalEmailSendingArgs
          }
      );
  }
  
  private static void ValidateEmailAddress(string emailAddress)
  {
      // here you should separate multiple address with "," and validate each one of them separately
      if(ValidationHelper.IsValidEmailAddress(emailAddress))
      {
          return;
      }

      throw new ArgumentException($"Email address '{emailAddress}' is not valid!");
  }
```
    
[ValidationHelper](https://github.com/abpframework/abp/blob/dev/framework/src/Volo.Abp.Validation.Abstractions/Volo/Abp/Validation/ValidationHelper.cs#L11)
    
```csharp
using System.Text.RegularExpressions;

namespace Volo.Abp.Validation;

public class ValidationHelper
{
    // Taken from W3C as an alternative to the RFC5322 specification: https://html.spec.whatwg.org/#valid-e-mail-address
    // The RFC5322 regex can be found here: https://emailregex.com/
    public static string EmailRegEx { get; set; } = @"^[a-zA-Z0-9.!#$%&'*+\/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$";

    public static bool IsValidEmailAddress(string email)
    {
        if (string.IsNullOrEmpty(email))
        {
            return false;
        }

       // this will validate for single address
        return Regex.IsMatch(email, EmailRegEx, RegexOptions.Compiled | RegexOptions.IgnoreCase);
    }
}
```

**Comment by maliming** [comment]
Thanks @ardenyy 

We will fix this in the next 9.0 patch version.

https://github.com/abpframework/abp/pull/21724
